packages/frontend/src/components/campaigns/index.tsx

```diff
@@ -145,6 +143,9 @@ export function Campaigns() {
     const [debouncedSearch, setDebouncedSearch] = useState(search);
     const [pageNumber, setPageNumber] = useState(1);
 
+    const data = useChainData(chain);
+    console.log("contract", data?.metromContract);
```

Comment:
Console log

Reply:
Addressed in 1e17d40cd8cf796c543abf3f7ecedc1bbe4b194b.